airflow/requirements.txt

```diff
@@ -1,4 +1,4 @@
-apache-airflow-providers-weaviate==1.0.0
+apache-airflow-providers-weaviate==1.3.0
```

Comment:
Why are we imposing this restriction? Will it always result in the installation of version 1.3.0?
can we just add  apache-airflow-providers-weaviate?

Reply:
We have a version for every library as you can see in the file. Are you recommending to remove version for all of them?

Reply:
For application (in this case), I think pinning it to some version is suggested.